src/ui/server/controllers/insurance/eligibility/buyer-country/index.ts

```diff
@@ -91,12 +91,12 @@ export const post = async (req: Request, res: Response) => {
     }
 
     /**
-     * If a country has no insurance support and no short term cover,
-     * redirect to TALK_TO_AN_EXPORT_FINANCE_MANAGER_EXIT.
+     * If a country does not have online insurance support,
+     * redirect to a specific exit page.
      */
-    const noShortTermCover = !country.noInsuranceSupport && !country.shortTermCover;
+    if (country.noOnlineInsuranceSupport) {
+      console.info(`Country support - %s ${country.name} - no online insurance support available`);
```

Comment:
sorry it should be ` console.info('Country support - %s - no online insurance support available', country.name);`